packages/lambda/commands/lambda.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+'use strict';
+
+module.exports = function defineLambdaCommand(yargs) {
+  yargs.command('lambda', 'manage your lambda deployment', function(yargs) {
+    yargs
+      .usage('Usage: $0 lambda <command>')
```

Comment:
As this thing might also be called locally/directly (i.e. using `npx`), I think we ought to use `hops-lambda` here instead of `$0`. See also: https://github.com/xing/hops/blob/master/packages/graphql/commands/graphql.js#L7